scripts/build/builders/qpg.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+import logging
+import os
+
+from .builder import Builder
+
+
+class QpgBuilder(Builder):
```

Comment:
Why does this need its own class? It's just a standard gn gen + ninja combination.

Reply:
This needs some consolidation once we have some more complete coverage of examples.

Agree that standard gn + ninja should be unified, maybe with some paths. I would prefer to do that later, especially if I figure out a way to not waste compilation and space for thins that can be a single 'gn gen'